dynamics/third_body_dynamics.py

```diff
@@ -0,0 +1,61 @@
+"""
+Module for computing third body dynamics and their Jacobians
+"""
+
+import brahe
+import numpy as np
+from brahe import GM_MOON, GM_SUN, Epoch
+
+
+def third_body_acceleration(x: np.ndarray, r_body: np.ndarray, mu: float) -> np.ndarray:
+    """
+    Computes the third body acceleration.
+
+    :param x: state vector
```

Comment:
Can you rename this to `r_sat` or something? There's no need for knowledge of the full state vector here. Same goes for the rest of the functions in this file.